hooks/environment

```diff
@@ -7,6 +7,10 @@ if [[ -z "${BUILDKITE_PLUGIN_AWS_ASSUME_ROLE_WITH_WEB_IDENTITY_ROLE_ARN:-}" ]];
   exit 1
 fi
 
+echo "--- :buildkite::key::aws: Requesting an OIDC token for AWS from buildkite"
+
+BUILDKITE_OIDC_TOKEN=$(buildkite-agent oidc request-token --audience sts.amazonaws.com)
```

Comment:
I reckon this should be wrapped in quotes, to be sure nothing strange happens with escaping:

```suggestion
BUILDKITE_OIDC_TOKEN="$(buildkite-agent oidc request-token --audience sts.amazonaws.com)"
```

Reply:
I, too, would have thought so, but my shellcheck didn't warn on it. Apparently it is not necessary? https://stackoverflow.com/a/71444610/5952021

Demo:
![demo](https://user-images.githubusercontent.com/11096602/202362969-54718966-f6b4-462f-966e-032a7c579c06.gif)
I'll accept the suggestion anyway as it doesn't hurt.